doc/conf.py

```diff
@@ -40,7 +40,8 @@
     'sphinxcontrib.napoleon',
     'nbsphinx',
     'IPython.sphinxext.ipython_console_highlighting',
-    'myst_parser'
+    'myst_parser',
+    'sphinx_panels'
```

Comment:
you also need to add this dependency to `setup.cfg`, in the [doc](https://github.com/dib-lab/sourmash/blob/5b698a656803390c47a20ae38d9a6e53a67974e1/setup.cfg#L66) section

Reply:
The package name is `sphinx-panels`, according to https://sphinx-panels.readthedocs.io/en/latest/#installation